public/car.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="98" height="98" viewBox="0 0 98 98" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M30.0719 24.5C27.755 24.5 25.9825 25.7536 25.3588 27.2089L19.957 39.8125H78.0431L72.6414 27.209C72.0176 25.7536 70.2451 24.5 67.9282 24.5H30.0719ZM85.69 42.2684C85.6435 42.0372 85.571 41.8154 85.4757 41.6062L78.2711 24.7962C76.5405 20.7581 72.3143 18.375 67.9282 18.375H30.0719C25.6858 18.375 21.4596 20.7581 19.729 24.7961C19.729 24.7961 19.729 24.7961 19.729 24.7961L12.5244 41.6063C12.4303 41.8127 12.3585 42.0313 12.312 42.2592C12.265 42.4868 12.2449 42.716 12.2501 42.9427V76.5625C12.2501 78.2539 13.6212 79.625 15.3126 79.625H21.4376C23.1289 79.625 24.5001 78.2539 24.5001 76.5625V73.5H73.5001V76.5625C73.5001 78.2539 74.8712 79.625 76.5626 79.625H82.6876C84.3789 79.625 85.7501 78.2539 85.7501 76.5625V42.9426C85.7551 42.719 85.7356 42.493 85.69 42.2684ZM79.6251 45.9375H18.3751V67.375H79.6251V45.9375Z" fill="#FFD053"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M21.4375 55.125C21.4375 51.7423 24.1798 49 27.5625 49C30.9452 49 33.6875 51.7423 33.6875 55.125C33.6875 58.5077 30.9452 61.25 27.5625 61.25C24.1798 61.25 21.4375 58.5077 21.4375 55.125Z" fill="#FFD053"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M64.3125 55.125C64.3125 51.7423 67.0548 49 70.4375 49C73.8202 49 76.5625 51.7423 76.5625 55.125C76.5625 58.5077 73.8202 61.25 70.4375 61.25C67.0548 61.25 64.3125 58.5077 64.3125 55.125Z" fill="#FFD053"/>
```

Comment:
change 'fill-rule' and 'clip-rule' to 'fillRule' and 'clipRule', respectively, or React might get mad at us

Reply:
when i change the svg to this, it fills in the car completely